2013-11-29 09:16:09,830 - [INFO] - from play in pool-4-thread-4 
Listening for HTTP on /0.0.0.0:9000

2013-11-29 09:17:05,892 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:17:06,532 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2013-11-29 09:19:52,398 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:19:52,575 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2013-11-29 09:20:10,425 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:20:10,550 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2013-11-29 09:20:50,022 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:20:50,147 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2013-11-29 09:32:57,074 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:32:57,200 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2013-11-29 09:34:51,023 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:34:51,145 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2013-11-29 09:38:27,435 - [INFO] - from play in play-internal-execution-context-19 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:38:27,547 - [INFO] - from play in play-internal-execution-context-19 
Application started (Dev)

2013-11-29 09:43:50,402 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:43:50,522 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2013-11-29 09:45:03,482 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:45:03,596 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2013-11-29 09:52:59,042 - [INFO] - from play in play-internal-execution-context-29 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:52:59,152 - [INFO] - from play in play-internal-execution-context-29 
Application started (Dev)

2013-11-29 09:54:03,317 - [INFO] - from play in play-internal-execution-context-38 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:54:03,429 - [INFO] - from play in play-internal-execution-context-38 
Application started (Dev)

2013-11-29 09:57:23,777 - [INFO] - from play in play-internal-execution-context-45 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 09:57:23,891 - [INFO] - from play in play-internal-execution-context-45 
Application started (Dev)

2013-11-29 10:06:57,331 - [INFO] - from play in play-internal-execution-context-49 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 10:06:57,442 - [INFO] - from play in play-internal-execution-context-49 
Application started (Dev)

2013-11-29 10:14:51,893 - [INFO] - from play in play-internal-execution-context-55 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 10:14:52,001 - [INFO] - from play in play-internal-execution-context-55 
Application started (Dev)

2013-11-29 10:15:22,236 - [INFO] - from play in play-internal-execution-context-55 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 10:15:22,346 - [INFO] - from play in play-internal-execution-context-55 
Application started (Dev)

2013-11-29 11:12:37,604 - [INFO] - from play in play-internal-execution-context-75 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:12:37,716 - [INFO] - from play in play-internal-execution-context-75 
Application started (Dev)

2013-11-29 11:13:29,874 - [INFO] - from play in play-internal-execution-context-75 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:13:29,984 - [INFO] - from play in play-internal-execution-context-75 
Application started (Dev)

2013-11-29 11:22:14,036 - [INFO] - from play in play-internal-execution-context-83 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:22:14,147 - [INFO] - from play in play-internal-execution-context-83 
Application started (Dev)

2013-11-29 11:28:48,202 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:28:48,317 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2013-11-29 11:34:50,356 - [INFO] - from play in play-internal-execution-context-86 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:34:50,466 - [INFO] - from play in play-internal-execution-context-86 
Application started (Dev)

2013-11-29 11:40:38,965 - [INFO] - from play in play-internal-execution-context-93 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:40:39,089 - [ERROR] - from application in New I/O worker #1 


! @6gdhfjh9m - Internal server error, for (GET) [/developer/index] ->

play.api.db.evolutions.InvalidDatabaseRevision: Database 'default' needs evolution![An SQL script need to be run on your database.]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1$$anonfun$apply$1.apply$mcV$sp(Evolutions.scala:486) ~[play-jdbc_2.10.jar:2.2.1]
	at play.api.db.evolutions.EvolutionsPlugin.withLock(Evolutions.scala:507) ~[play-jdbc_2.10.jar:2.2.1]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:461) ~[play-jdbc_2.10.jar:2.2.1]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:459) ~[play-jdbc_2.10.jar:2.2.1]
	at scala.collection.immutable.List.foreach(List.scala:318) ~[scala-library.jar:na]
	at play.api.db.evolutions.EvolutionsPlugin.onStart(Evolutions.scala:459) ~[play-jdbc_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at scala.collection.immutable.List.foreach(List.scala:318) ~[scala-library.jar:na]
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18) ~[play_2.10.jar:2.2.1]
	at play.api.Play$.start(Play.scala:87) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.1]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.1]
	at scala.util.Success.flatMap(Try.scala:200) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.1]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2013-11-29 11:40:42,055 - [INFO] - from play in play-internal-execution-context-93 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:40:42,243 - [ERROR] - from application in New I/O worker #1 


! @6gdhfjm18 - Internal server error, for (GET) [/developer/index] ->

play.api.UnexpectedException: Unexpected exception[ConstructorException: null; Can't construct a java object for tag:yaml.org,2002:models.CategoryModel; exception=Cannot create property=type for JavaBean=models.CategoryModel@1; Unable to find enum value '0' for enum class: models.dto.CategoryType;  in 'reader', line 5, column 7:
        - !!models.CategoryModel
          ^]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:148) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.1]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.1]
	at scala.util.Success.flatMap(Try.scala:200) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.1]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]
Caused by: org.yaml.snakeyaml.constructor.ConstructorException: null; Can't construct a java object for tag:yaml.org,2002:models.CategoryModel; exception=Cannot create property=type for JavaBean=models.CategoryModel@1; Unable to find enum value '0' for enum class: models.dto.CategoryType;  in 'reader', line 5, column 7:
        - !!models.CategoryModel
          ^
	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:333) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructSequenceStep2(BaseConstructor.java:276) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructSequence(BaseConstructor.java:247) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.SafeConstructor$ConstructYamlSeq.construct(SafeConstructor.java:442) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructMapping2ndStep(BaseConstructor.java:325) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.SafeConstructor.constructMapping2ndStep(SafeConstructor.java:145) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructMapping(BaseConstructor.java:306) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.SafeConstructor$ConstructYamlMap.construct(SafeConstructor.java:461) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructDocument(BaseConstructor.java:141) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:127) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:481) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.Yaml.load(Yaml.java:412) ~[snakeyaml.jar:na]
	at play.libs.Yaml.load(Yaml.java:31) ~[play-java_2.10.jar:2.2.1]
	at play.libs.Yaml.load(Yaml.java:18) ~[play-java_2.10.jar:2.2.1]
	at Global$InitialData.insert(Global.java:34) ~[na:na]
	at Global.onStart(Global.java:25) ~[na:na]
	at play.core.j.JavaGlobalSettingsAdapter.onStart(JavaGlobalSettingsAdapter.scala:18) ~[play_2.10.jar:2.2.1]
	at play.api.GlobalPlugin.onStart(GlobalSettings.scala:203) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at scala.collection.immutable.List.foreach(List.scala:318) ~[scala-library.jar:na]
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.1]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18) ~[play_2.10.jar:2.2.1]
	at play.api.Play$.start(Play.scala:87) ~[play_2.10.jar:2.2.1]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139) ~[play_2.10.jar:2.2.1]
	... 14 common frames omitted
Caused by: org.yaml.snakeyaml.error.YAMLException: Cannot create property=type for JavaBean=models.CategoryModel@1; Unable to find enum value '0' for enum class: models.dto.CategoryType
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.constructJavaBean2ndStep(Constructor.java:299) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.construct(Constructor.java:189) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:331) ~[snakeyaml.jar:na]
	... 43 common frames omitted
Caused by: org.yaml.snakeyaml.error.YAMLException: Unable to find enum value '0' for enum class: models.dto.CategoryType
	at org.yaml.snakeyaml.constructor.Constructor$ConstructScalar.constructStandardJavaInstance(Constructor.java:477) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.Constructor$ConstructScalar.construct(Constructor.java:365) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182) ~[snakeyaml.jar:na]
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.constructJavaBean2ndStep(Constructor.java:296) ~[snakeyaml.jar:na]
	... 45 common frames omitted

2013-11-29 11:42:55,455 - [INFO] - from play in play-internal-execution-context-93 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:42:55,597 - [INFO] - from play in play-internal-execution-context-93 
Application started (Dev)

2013-11-29 11:43:25,099 - [INFO] - from play in play-internal-execution-context-95 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:43:25,210 - [INFO] - from play in play-internal-execution-context-95 
Application started (Dev)

2013-11-29 11:51:54,994 - [INFO] - from play in play-internal-execution-context-97 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 11:51:55,103 - [INFO] - from play in play-internal-execution-context-97 
Application started (Dev)

2013-11-29 14:26:34,833 - [INFO] - from play in play-internal-execution-context-101 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:26:34,951 - [INFO] - from play in play-internal-execution-context-101 
Application started (Dev)

2013-11-29 14:26:35,442 - [ERROR] - from application in New I/O worker #3 


! Internal server error, for (GET) [/apps/category/[object%20Object]/1] ->

java.net.URISyntaxException: Illegal character in path at index 15: /apps/category/[object%20Object]/1
	at java.net.URI$Parser.fail(Unknown Source) ~[na:1.6.0_43]
	at java.net.URI$Parser.checkChars(Unknown Source) ~[na:1.6.0_43]
	at java.net.URI$Parser.parseHierarchical(Unknown Source) ~[na:1.6.0_43]
	at java.net.URI$Parser.parse(Unknown Source) ~[na:1.6.0_43]
	at java.net.URI.<init>(Unknown Source) ~[na:1.6.0_43]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anon$1.path(PlayDefaultUpstreamHandler.scala:98) ~[play_2.10.jar:2.2.1]
	at play.docs.DocumentationHandler.maybeHandleDocRequest(DocumentationHandler.scala:58) ~[na:na]
	at play.docs.DocumentationHandler.maybeHandleDocRequest(DocumentationHandler.scala:28) ~[na:na]
	at play.core.ReloadableApplication.handleWebCommand(ApplicationProvider.scala:171) ~[play_2.10.jar:2.2.1]
	at play.core.server.Server$$anonfun$getHandlerFor$1.apply(Server.scala:79) ~[play_2.10.jar:2.2.1]
	at play.core.server.Server$$anonfun$getHandlerFor$1.apply(Server.scala:79) ~[play_2.10.jar:2.2.1]
	at scala.util.control.Exception$Catch$$anonfun$either$1.apply(Exception.scala:124) ~[scala-library.jar:na]
	at scala.util.control.Exception$Catch$$anonfun$either$1.apply(Exception.scala:124) ~[scala-library.jar:na]
	at scala.util.control.Exception$Catch.apply(Exception.scala:102) ~[scala-library.jar:na]
	at scala.util.control.Exception$Catch.either(Exception.scala:124) ~[scala-library.jar:na]
	at play.core.server.Server$class.getHandlerFor(Server.scala:79) ~[play_2.10.jar:2.2.1]
	at play.core.server.NettyServer.getHandlerFor(NettyServer.scala:35) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$8.apply(PlayDefaultUpstreamHandler.scala:117) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$8.apply(PlayDefaultUpstreamHandler.scala:117) [play_2.10.jar:2.2.1]
	at scala.util.Either.fold(Either.scala:98) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler.messageReceived(PlayDefaultUpstreamHandler.scala:111) [play_2.10.jar:2.2.1]
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791) [netty.jar:na]
	at com.typesafe.netty.http.pipelining.HttpPipeliningHandler.messageReceived(HttpPipeliningHandler.java:62) [netty-http-pipelining.jar:na]
	at org.jboss.netty.channel.SimpleChannelHandler.handleUpstream(SimpleChannelHandler.java:88) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791) [netty.jar:na]
	at org.jboss.netty.handler.codec.http.HttpContentDecoder.messageReceived(HttpContentDecoder.java:108) [netty.jar:na]
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791) [netty.jar:na]
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:296) [netty.jar:na]
	at org.jboss.netty.handler.codec.frame.FrameDecoder.unfoldAndFireMessageReceived(FrameDecoder.java:459) [netty.jar:na]
	at org.jboss.netty.handler.codec.replay.ReplayingDecoder.callDecode(ReplayingDecoder.java:536) [netty.jar:na]
	at org.jboss.netty.handler.codec.replay.ReplayingDecoder.messageReceived(ReplayingDecoder.java:435) [netty.jar:na]
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:559) [netty.jar:na]
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:268) [netty.jar:na]
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:255) [netty.jar:na]
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:88) [netty.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:109) [netty.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312) [netty.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:90) [netty.jar:na]
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178) [netty.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) [netty.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) [netty.jar:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source) [na:1.6.0_43]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.6.0_43]
	at java.lang.Thread.run(Unknown Source) [na:1.6.0_43]

2013-11-29 14:34:36,667 - [INFO] - from play in play-internal-execution-context-104 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:34:36,788 - [INFO] - from play in play-internal-execution-context-104 
Application started (Dev)

2013-11-29 14:38:23,252 - [INFO] - from play in play-internal-execution-context-104 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:38:23,362 - [INFO] - from play in play-internal-execution-context-104 
Application started (Dev)

2013-11-29 14:46:53,824 - [INFO] - from play in play-internal-execution-context-109 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:46:53,933 - [INFO] - from play in play-internal-execution-context-109 
Application started (Dev)

2013-11-29 14:49:35,902 - [INFO] - from play in play-internal-execution-context-109 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:49:36,016 - [INFO] - from play in play-internal-execution-context-109 
Application started (Dev)

2013-11-29 14:50:11,252 - [INFO] - from play in play-internal-execution-context-115 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 14:50:11,361 - [INFO] - from play in play-internal-execution-context-115 
Application started (Dev)

2013-11-29 14:53:54,120 - [ERROR] - from play in play-internal-execution-context-126 
Cannot invoke the action, eventually got an error: java.lang.NullPointerException

2013-11-29 14:53:54,134 - [ERROR] - from application in play-internal-execution-context-126 


! @6gdif39l6 - Internal server error, for (GET) [/apps/category/1/1] ->

play.api.Application$$anon$1: Execution exception[[NullPointerException: null]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.1]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10.jar:2.2.1]
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:33) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:411) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:408) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.1]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:37) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:133) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:58) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:254) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:249) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.NullPointerException: null
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:65) ~[na:na]
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:87) ~[na:na]
	at models.AppModel.findAppsByCategory(AppModel.java:120) ~[na:na]
	at controllers.AppsShow.getCategoryapps(AppsShow.java:46) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.1]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.1]
	at play.GlobalSettings$1.call(GlobalSettings.java:64) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:278) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:274) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.1]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted

2013-11-29 14:54:52,403 - [ERROR] - from play in play-internal-execution-context-127 
Cannot invoke the action, eventually got an error: java.lang.NullPointerException

2013-11-29 14:54:52,408 - [ERROR] - from application in play-internal-execution-context-127 


! @6gdif6i0f - Internal server error, for (GET) [/apps/category/1/1] ->

play.api.Application$$anon$1: Execution exception[[NullPointerException: null]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.1]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10.jar:2.2.1]
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:33) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:411) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:408) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.1]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:37) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:133) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:58) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:254) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:249) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.NullPointerException: null
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:65) ~[na:na]
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:87) ~[na:na]
	at models.AppModel.findAppsByCategory(AppModel.java:120) ~[na:na]
	at controllers.AppsShow.getCategoryapps(AppsShow.java:46) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.1]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.1]
	at play.GlobalSettings$1.call(GlobalSettings.java:64) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:278) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:274) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.1]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted

2013-11-29 15:01:48,145 - [INFO] - from play in play-internal-execution-context-127 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:01:48,253 - [INFO] - from play in play-internal-execution-context-127 
Application started (Dev)

2013-11-29 15:01:55,228 - [ERROR] - from play in play-internal-execution-context-131 
Cannot invoke the action, eventually got an error: java.lang.NullPointerException

2013-11-29 15:01:55,232 - [ERROR] - from application in play-internal-execution-context-131 


! @6gdig4jd1 - Internal server error, for (GET) [/apps/category/1/1] ->

play.api.Application$$anon$1: Execution exception[[NullPointerException: null]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.1]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10.jar:2.2.1]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10.jar:2.2.1]
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:33) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:411) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:408) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.1]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:37) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:133) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:58) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:254) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:249) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.NullPointerException: null
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:68) ~[na:na]
	at models.dto.SimpleAppModel.transferAM(SimpleAppModel.java:89) ~[na:na]
	at models.AppModel.findAppsByCategory(AppModel.java:120) ~[na:na]
	at controllers.AppsShow.getCategoryapps(AppsShow.java:46) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:245) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.1]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.1]
	at play.GlobalSettings$1.call(GlobalSettings.java:64) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.1]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.1]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:278) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:274) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:29) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.1]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted

2013-11-29 15:02:59,637 - [INFO] - from play in play-internal-execution-context-127 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:02:59,745 - [INFO] - from play in play-internal-execution-context-127 
Application started (Dev)

2013-11-29 15:18:13,683 - [INFO] - from play in play-internal-execution-context-127 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:18:13,793 - [INFO] - from play in play-internal-execution-context-127 
Application started (Dev)

2013-11-29 15:19:38,195 - [INFO] - from play in play-internal-execution-context-127 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:19:38,307 - [INFO] - from play in play-internal-execution-context-127 
Application started (Dev)

2013-11-29 15:27:31,140 - [INFO] - from play in play-internal-execution-context-139 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:27:31,248 - [INFO] - from play in play-internal-execution-context-139 
Application started (Dev)

2013-11-29 15:43:10,568 - [INFO] - from play in play-internal-execution-context-144 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:43:10,676 - [INFO] - from play in play-internal-execution-context-144 
Application started (Dev)

2013-11-29 15:45:22,505 - [INFO] - from play in play-internal-execution-context-149 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:45:22,614 - [INFO] - from play in play-internal-execution-context-149 
Application started (Dev)

2013-11-29 15:46:52,786 - [INFO] - from play in play-internal-execution-context-153 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:46:52,893 - [INFO] - from play in play-internal-execution-context-153 
Application started (Dev)

2013-11-29 15:47:50,926 - [INFO] - from play in play-internal-execution-context-153 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:47:51,035 - [INFO] - from play in play-internal-execution-context-153 
Application started (Dev)

2013-11-29 15:57:19,703 - [INFO] - from play in play-internal-execution-context-162 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:57:19,811 - [INFO] - from play in play-internal-execution-context-162 
Application started (Dev)

2013-11-29 15:57:59,105 - [INFO] - from play in play-internal-execution-context-164 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 15:57:59,211 - [INFO] - from play in play-internal-execution-context-164 
Application started (Dev)

2013-11-29 16:00:03,801 - [INFO] - from play in play-internal-execution-context-165 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:00:03,944 - [INFO] - from play in play-internal-execution-context-165 
Application started (Dev)

2013-11-29 16:00:54,127 - [INFO] - from play in play-internal-execution-context-168 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:00:54,234 - [INFO] - from play in play-internal-execution-context-168 
Application started (Dev)

2013-11-29 16:01:07,834 - [INFO] - from play in play-internal-execution-context-170 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:01:07,939 - [INFO] - from play in play-internal-execution-context-170 
Application started (Dev)

2013-11-29 16:02:08,058 - [INFO] - from play in play-internal-execution-context-173 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:02:08,166 - [INFO] - from play in play-internal-execution-context-173 
Application started (Dev)

2013-11-29 16:03:16,248 - [INFO] - from play in play-internal-execution-context-174 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:03:16,360 - [INFO] - from play in play-internal-execution-context-174 
Application started (Dev)

2013-11-29 16:15:16,309 - [INFO] - from play in play-internal-execution-context-180 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:15:16,421 - [INFO] - from play in play-internal-execution-context-180 
Application started (Dev)

2013-11-29 16:16:25,589 - [INFO] - from play in play-internal-execution-context-184 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:16:25,697 - [INFO] - from play in play-internal-execution-context-184 
Application started (Dev)

2013-11-29 16:17:28,401 - [INFO] - from play in play-internal-execution-context-189 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:17:28,510 - [INFO] - from play in play-internal-execution-context-189 
Application started (Dev)

2013-11-29 16:18:30,624 - [INFO] - from play in play-internal-execution-context-196 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:18:30,731 - [INFO] - from play in play-internal-execution-context-196 
Application started (Dev)

2013-11-29 16:21:10,483 - [INFO] - from play in play-internal-execution-context-198 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:21:10,590 - [INFO] - from play in play-internal-execution-context-198 
Application started (Dev)

2013-11-29 16:23:06,053 - [INFO] - from play in play-internal-execution-context-209 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:23:06,160 - [INFO] - from play in play-internal-execution-context-209 
Application started (Dev)

2013-11-29 16:24:44,932 - [INFO] - from play in play-internal-execution-context-217 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:24:45,039 - [INFO] - from play in play-internal-execution-context-217 
Application started (Dev)

2013-11-29 16:50:05,272 - [INFO] - from play in play-internal-execution-context-230 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:50:05,382 - [INFO] - from play in play-internal-execution-context-230 
Application started (Dev)

2013-11-29 16:53:41,154 - [INFO] - from play in play-internal-execution-context-245 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 16:53:41,263 - [INFO] - from play in play-internal-execution-context-245 
Application started (Dev)

2013-11-29 17:55:10,190 - [INFO] - from play in play-internal-execution-context-315 
database [default] connected at jdbc:mysql://localhost/market?useUnicode=true&characterEncoding=UTF-8

2013-11-29 17:55:10,298 - [INFO] - from play in play-internal-execution-context-315 
Application started (Dev)

